interface/forms/dictation/report.php

```diff
@@ -18,8 +39,8 @@ function dictation_report( $pid, $encounter, $cols, $id) {
         $value = "yes";
       }
       $key=ucwords(str_replace("_"," ",$key));
-      print "<td><span class=bold>" . xlt($key) . ": </span><span class=text>" .
-        nl2br(text($value)) . "</span></td>";
+      print "<td><span class='bold'>" . xlt($key) . ": </span><span class='text'>" .
+        html_entity_decode($value, ENT_QUOTES | ENT_HTML5) . "</span></td>";
```

Comment:
I was doing some reading and it is not the job of the editor but the server side store process. So, reading that, I changed the way the data is stored. I put the input filters in place and then converted any HTML into entities and that is stored in the tables. Now we should be safe from XSS attacks.